LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2022 Canonical Ltd.
+   Copyright 2023 Canonical Ltd.
```

Comment:
Do we need to add `Omnivector, LLC` to this License too?

Reply:
To be honest, that question is above me :sweat_smile:

Let me check with the relevant parties. If we need to add `Omnivector, LLC` to the license, this should be changed in the other slurm operators. I believe all the LICENSE files originally had "_Copyright 2022 Canonical Ltd._" so they will need to updated.

Reply:
This is for the license file, and I believe that it was in error to define the license with the copyright assignment to Canonical. We can dual assign the copyright, but from my perspective these are the omnivector slurm charms and the copyright is still Omnivector's.

That being said, new files introduced by Canonical may contain the copyright assignment to Canonical rather than Omnivector. This is standard in projects such as those found in the OpenStack big-tent projects.

Reply:
Sounds good to me. Thanks!

Reply:
```suggestion
   Copyright 2023 Omnivector, LLC
```

Reply:
I updated the author using GitHub suggestions. As such, I have marked this conversation as resolved.